services/order.py

```diff
@@ -0,0 +1,28 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None) -> Order:
+    order = Order.objects.create(user=User.objects.get(username=username))
+    if date is not None:
+        order.created_at = date
+        order.save()
+    for ticket_data in tickets:
+        Ticket.objects.create(
+            order=order,
+            movie_session_id=ticket_data["movie_session"],
+            row=ticket_data["row"],
+            seat=ticket_data["seat"])
+    return order
+
+
+def get_orders(username: str = None) -> list[Order]:
```

Comment:
The return type annotation `list[Order]` is not compatible with Python versions below 3.9. Consider using `List[Order]` from the `typing` module for better compatibility.